src/GovernorCountingFractional.sol

```diff
@@ -134,23 +146,28 @@ abstract contract GovernorCountingFractional is Governor {
      */
     function _countVoteFractional(
         uint256 proposalId,
-        uint256 weight,
+        address account,
+        uint128 weight,
         bytes memory voteData
     ) internal {
         require(voteData.length == 48, "GovernorCountingFractional: invalid voteData");
 
-        (uint128 forVotes, uint128 againstVotes, uint128 abstainVotes) = _decodePackedVotes(voteData);
+        (uint128 _forVotes, uint128 _againstVotes, uint128 _abstainVotes) = _decodePackedVotes(voteData);
```

Comment:
_(arbitrary comment location to start a thread)_

One small comment is that this PR would have been easier to review without the variable name changes (i.e. adding a leading underscore), since lots of lines had changes like this so I had to see if that was the only change on the line or not. Ideally we should handle things like that in their own PR to keep diffs simpler, not a big deal though of course 🙂

Reply:
Noted. Though I'll be honest that I made all of these changes because I expected to get comments on it if I didn't update the old code to fit the current style guide

Reply:
Personally I don't like the leading underscore convention, so you wouldn't have gotten comments from me 😛 (I find it adds a lot of noise and hurts readability without adding much value, and there's other ways to solve the the same problem anyway)

Reply:
Either way, resolving 👌